File: codes/cpp/inheritance/single/person.cpp
#include "person.h"

Person::Person(std::string name)
{
    this->name = name;
    this->luckyNumber = 2;
}

void Person::info()
{
    std::cout << "My name: " << this->name << "\n";
    std::cout << "My Lucky number: " << this->luckyNumber << "\n";
}

File: codes/java/src/inheritance/single/Person.java
package inheritance.single;

public class Person {

    public int luckyNumber;
    private String name;

    public Person(String name) {
        this.name = name;
        luckyNumber = 0;
    }

    public void info() {
        System.out.println("My name: " + name);
        System.out.println("My lucky number is: " + luckyNumber);
    }

    private void info2() {
        System.out.println("Person.info2");
    }

}
